lib/astarte/client/appengine.ex

```diff
@@ -17,18 +17,69 @@
 #
 
 defmodule Astarte.Client.AppEngine do
+  @moduledoc """
+  Module to configure communication with Astarte AppEngine API.
+
+  Most applications would want to use AppEngine to interact with devices, stream and receive data, and oversee their fleet.
+  """
+  @moduledoc since: "0.1.0"
+
+  alias Astarte.Client.Credentials
+
+  @jwt_expiry 5 * 60
   @enforce_keys [:http_client]
+
   defstruct @enforce_keys
 
   @type t :: %__MODULE__{
           http_client: Tesla.Client.t()
         }
 
-  alias __MODULE__
-  alias Astarte.Client.Credentials
+  @doc """
+  Returns configured `Astarte.Client.AppEngine` struct.
 
-  @jwt_expiry 5 * 60
+  This function receives Astarte base API URL and name of the realm alongside authentication options
+  and returns configured `Astarte.Client.AppEngine` struct.
```

Comment:
Stylistic and writing suggestions:

and returns **the** configured `Astarte.Client.AppEngine` struct.